e2e/src/test-environment/orbs-nodes.ts

```diff
@@ -49,7 +49,7 @@ export class OrbsNode implements TestComponent {
       command += " --forceRecreate";
     }
     await this.runDockerCompose(command);
-    await delay(60000);
+    // await delay(60000);
```

Comment:
why was this removed? if not required remove comment

Reply:
Replaced by actual logic that proceeds with the test once all docker containers are healthy, instead of waiting arbitrary time. Comment removed.